corehq/apps/importer/util.py

```diff
@@ -404,12 +410,16 @@ def populate_updated_fields(config, columns, row, datemode):
 
         if update_value is not None:
             if field_map[key]['type_field'] == 'date':
-                update_value = parse_excel_date(update_value, datemode)
-            elif field_map[key]['type_field'] == 'integer':
                 try:
-                    update_value = str(int(update_value))
-                except ValueError:
-                    update_value = ''
```

Comment:
Previously, if your value was not an integer, but you thought it should be, we would just remove it silently. 
:kimono: